docs/essentials/includes/get-started.md

```diff
@@ -1,7 +1,5 @@
 ---
 ms.topic: include
-author: jamesmontemagno
-ms.author: jamont
-ms.date: 10/31/2018
+ms.date: 08/02/2021
 ---
-To start using this API, read the [getting started](~/essentials/get-started.md) guide for Xamarin.Essentials to ensure the library is properly installed and set up in your projects.
+To start using this API, read the [getting started](../get-started.md) guide for Xamarin.Essentials to ensure the library is properly installed and set up in your projects.
```

Comment:
1. .NET MAUI Essentials
2. Best not to call it a library anymore (Xamarin.Essentials was an additional library you had to install, now it's just part of MAUI).
3. It's best now to say something like see the getting started guide to ensure .NET MAUI Essentials is correctly setup/configured in your projects.